R/source.R

```diff
@@ -22,11 +22,26 @@ source_package <- function(path) {
   load_pkg_dependencies(path)
 
   paths <- package_files(path)
-  lapply(paths, sys.source, envir = env, keep.source = FALSE)
+  lapply(paths, sys_source, envir = env)
 
   env
 }
 
+sys_source <- function(file, envir = baseenv()) {
```

Comment:
If you copy code from base R, you must include an attribution of the source, check that the licenses are compatible, and add base R to the `Authors@R`. Instead, I think you can create a `file()` connection with the correct encoding and pass that to `sys.source()`